src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+  return !a;
+  //
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !a && !b;
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b) || (b&&!a);
 };
 
 function truthiness(a) {
-  // your code here
+  return !!a;
 };
 
 function isEqual(a, b) {
-  // your code here
+  return !!(a==b) ;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return (a>b);
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return (a<=b);
 };
 
 function isOdd(a) {
-  // your code here
+  return !((a%2)==0);
 };
 
 function isEven(a) {
-  // your code here
+  return ((a%2)==0);
 };
 
 function isSquare(a) {
-  // your code here
+  return a>=0&&Math.sqrt(a)%1===0;
 };
 
 function startsWith(char, string) {
-  // your code here
+  return string.startsWith(char);
 };
 
 function containsVowels(string) {
-  // your code here
-};
+  string.toLowerCase();
+  let m = string.match(/[aeiou]/gi);
+  return m === null ? false : true;
+}
 
 function isLowerCase(string) {
-  // your code here
-};
+  if ( string == string.toLowerCase()){
```

Comment:
We should always use `===` over `==` where possible.

Reply:
This is good and works, but a simpler way using the same logic would to `return string === string.toLowerCase()`